airflow/contrib/example_dags/example_gcp_vision.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that creates, gets, updates and deletes Products and Product Sets in the Google Cloud 
+Vision service in the Google Cloud Platform.
+
+This DAG relies on the following OS environment variables
+
+* GCP_VISION_LOCATION - Google Cloud Platform zone where the instance exists.
+"""
+import os
+
+from google.api_core.retry import Retry
+from google.cloud.vision_v1.proto.product_search_service_pb2 import ProductSet, Product
+
+import airflow
+from airflow import models
+from airflow.contrib.operators.gcp_vision_operator import CloudVisionProductSetCreateOperator, \
+    CloudVisionProductSetGetOperator, CloudVisionProductSetUpdateOperator, \
+    CloudVisionProductSetDeleteOperator, CloudVisionProductCreateOperator, CloudVisionProductGetOperator, \
+    CloudVisionProductUpdateOperator, CloudVisionProductDeleteOperator
+
```

Comment:
I personally prefer to use a different syntax when there are many classes.
```python
from airflow.contrib.operators.gcp_vision_operator import (
    CloudVisionProductSetCreateOperator, 
    CloudVisionProductSetGetOperator, 
    CloudVisionProductSetUpdateOperator, 
    CloudVisionProductSetDeleteOperator, 
    CloudVisionProductCreateOperator, 
    CloudVisionProductGetOperator, 
    CloudVisionProductUpdateOperator, 
    CloudVisionProductDeleteOperator
)
```
It is cleaner and works better with a git.
This is not a caveat, but a small suggestion.

Reply:
Yeah that's a problem, but I changed it anyway according to the suggestion as it does look much cleaner.